packages/hoppscotch-common/src/components/share/index.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <div
+      class="sticky top-0 z-10 flex flex-shrink-0 flex-col overflow-x-auto bg-primary"
+    >
+      <WorkspaceCurrent
+        :section="t('tab.shared_requests')"
+        :is-only-personal="true"
```

Comment:
The current workspace title shows up as `My workspace > Shared Requests` all the time. This is intentional, right? I noticed that shared requests are always associated with a user.

Reply:
yup this is intentional since shared requests are associated with a user only.